### ISSUE TITLE
dotnet fsi - tabbing two properties deep fails with FS1108

### ISSUE THREAD

**Issue by ChernayaKoshka** [dotnet/fsharp#10330]
When using tab-complete in an instance of `dotnet fsi`, it will kick out an FS1108 error related to the "Windows" assembly if you are tabbing two properties deep. This does _not_ prevent the actual usage of the second property as you can type it out and access it that way just fine.

Screenshots for:
* Alpine Linux
![Alpine Linux](https://user-images.githubusercontent.com/8130999/97049107-fb60bf80-152f-11eb-8e94-7b3d3b33acc3.png)

* Windows 10 Pro
![image](https://user-images.githubusercontent.com/8130999/97049165-129fad00-1530-11eb-9da2-bf007164b90a.png)

**Repro steps**

Provide the steps required to reproduce the problem:

1. Start FSI using `dotnet fsi`
2. Find an object with two properties you need to tab into (ie, DateTime.Now.Ticks or CultureInfo.CurrentCulture.Name)
3. Press tab to tab complete after the first property

**Expected behavior**

FSI would complete or tab through available options. Such as `Ticks` or `Name`.

**Actual behavior**

In all cases, it will fail with error FS1108, the exact nature of the error seems to vary between my work laptop and my home computer / Alpine Linux box.

Work Laptop Error: ``stdin(0,1): error FS1108: The type 'IRandomAccessStream' is required here and is unavailable. You must add a reference to assembly 'Windows, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null``

Home PC Error: ``stdin(0,1): error FS1108: The type 'IAsyncOperationWithProgress`2' is required here and is unavailable. You must add a reference to assembly 'Windows, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null'.``

Alpine Linux Error: ``/root/stdin(0,1): error FS1108: The type 'IAsyncOperationWithProgress`2' is required here and is unavailable. You must add a reference to assembly 'Windows, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null'.``

**Known workarounds**

* Not tab-completing and just typing out the name of the property works fine.
* For Windows: Using the non-core FSI version (fsi.exe).

**Related information**

Provide any related information (optional):

**Home PC**:
* Windows 10 Pro, Version 10.19041 Build 19041
* Installed Runtimes:
```
Microsoft.AspNetCore.All 2.1.23
Microsoft.AspNetCore.App 2.1.23
Microsoft.AspNetCore.App 3.1.1
Microsoft.AspNetCore.App 3.1.9
Microsoft.NETCore.App 2.1.23
Microsoft.NETCore.App 3.1.1
Microsoft.NETCore.App 3.1.9
Microsoft.WindowsDesktop.App 3.1.1
Microsoft.WindowsDesktop.App 3.1.9
```

**Work PC**:
* Windows 10 Pro, Version 10.0.17763 Build 17763
* Installed Runtimes:
```
Microsoft.AspNetCore.All 2.1.11
Microsoft.AspNetCore.All 2.1.19
Microsoft.AspNetCore.All 2.1.22
Microsoft.AspNetCore.All 2.2.5
Microsoft.AspNetCore.All 2.2.8
Microsoft.AspNetCore.App 2.1.11
Microsoft.AspNetCore.App 2.1.19
Microsoft.AspNetCore.App 2.1.22
Microsoft.AspNetCore.App 2.2.5
Microsoft.AspNetCore.App 2.2.8
Microsoft.AspNetCore.App 3.1.1
Microsoft.AspNetCore.App 3.1.5
Microsoft.AspNetCore.App 3.1.8
Microsoft.NETCore.App 2.1.11
Microsoft.NETCore.App 2.1.19
Microsoft.NETCore.App 2.1.22
Microsoft.NETCore.App 2.2.5
Microsoft.NETCore.App 2.2.8
Microsoft.NETCore.App 3.1.5
Microsoft.NETCore.App 3.1.8
Microsoft.WindowsDesktop.App 3.1.5
Microsoft.WindowsDesktop.App 3.1.8
```

**Alpine Linux Box**:
* Alpine Linux v3.12
* Installed Runtimes:
```
Microsoft.AspNetCore.App 3.1.9
Microsoft.NETCore.App 3.1.9
```

**Comment by Swoorup** [comment]
Same as https://github.com/dotnet/fsharp/issues/9206

**Comment by ChernayaKoshka** [comment]
I searched everywhere to see if this had already been reported. I guess my Google-fu is lacking. Given the issue was fixed in #9644, I'm going ahead and closing this.

**Comment by ChernayaKoshka** [comment]
I searched everywhere to see if this had already been reported. I guess my Google-fu is lacking. Given the issue was fixed in #9644, I'm going ahead and closing this.
